blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnableDrainExecutionContextTest.java

```diff
@@ -17,6 +24,10 @@
 
 import static org.mockito.Mockito.*;
 
+@PowerMockIgnore({"javax.management.*", "com.rackspacecloud.blueflood.utils.Metrics", "com.codahale.metrics.*"})
+@PrepareForTest({ IOContainer.class })
+@RunWith(PowerMockRunner.class)
+@SuppressStaticInitializationFor( "com.rackspacecloud.blueflood.io.IOContainer" )
```

Comment:
@usnavi 
Yes if we still want to truly test verifyZeroInteractions(locatorIO);

But looking at these tests it seems a bit bloated with all these verifyZeroInteractions, so we can just get rid of the assert for verifyZeroInteractions(locatorIO); and we wouldn't need to set up the static mock (because of the change in LocatorFetchRunnable).